packages/integration-sdk-runtime/src/execution/executeIntegration.ts

```diff
@@ -218,96 +214,116 @@ export async function executeWithContext<
   }
 
   try {
-    await removeStorageDirectory();
+    if (shouldPublishDiskUsageMetric) {
+      await tryPublishDiskUsageMetric(context);
+    }
```

Comment:
Why does this exist when there is just the interval maker above?

Reply:
Spoke in DM: I think this code can be removed and was likely originally put here to establish a baseline. We discussed that the code can probably be re-organized a bit, but removing this entirely may be the best option.